tests/template/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -63,17 +63,19 @@ g
 }\`;
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 \`
-(?:\${escapeChar}[\\\\S\\\\s]|(?:(?!\${// Using \`XRegExp.union\` safely rewrites backreferences in \`left\` and \`right\`.
-// Intentionally not passing \`basicFlags\` to \`XRegExp.union\` since any syntax
-// transformation resulting from those flags was already applied to \`left\` and
-// \`right\` when they were passed through the XRegExp constructor above.
-XRegExp.union([left, right], "", { conjunction: "or" })
-  .source})[^\${escapeChar}])+)+
+(?:\${escapeChar}[\\\\S\\\\s]|(?:(?!\${
+  // Using \`XRegExp.union\` safely rewrites backreferences in \`left\` and \`right\`.
+  // Intentionally not passing \`basicFlags\` to \`XRegExp.union\` since any syntax
+  // transformation resulting from those flags was already applied to \`left\` and
+  // \`right\` when they were passed through the XRegExp constructor above.
+  XRegExp.union([left, right], "", { conjunction: "or" }).source
+})[^\${escapeChar}])+)+
```

Comment:
... but this is 100x better than it was before.

Perhaps we could differentiate single comment vs. multiple comments? Or another heuristic..